site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1771,6 +1773,20 @@ components:
             value: "{\"field\":\"/ext/brand\"}"
           sort descending:
             value: "{\"field\":\"/status/created\",\"direction\":\"desc\"}"
+      isGateway:
+        name: isGateway
+        in: query
+        description: |
+          Boolean filter parameter for searching only devices or gateways.
```

Comment:
what if I do not specify the query parameter at all?

Reply:
Then we'll get both devices and gateways (Hono Default search functionality)

Reply:
My point is that IMHO we should document this explicitly ...

Reply:
Done